src/components/organisms/cards/cards.scss

```
@@ -0,0 +1,33 @@
+@use "../../variables" as *;
+
+.container {
+  padding: 0px;
+  margin: 0px;
+  .cards {
+    display: flex;
+    justify-content: space-between;
+    padding: 0px 0px 100px;
+    gap: 24px;
+    @media (min-width: $breakpoint-lg) {
+      display: block;
+      padding-bottom: 40px;
+    }
+  }
+  .card {
+    flex: 1;
+    .card-body {
+      .third {
+        @media (max-width: $breakpoint-lg) {
+          margin-bottom: 63px;
+        }
+      }
```

```suggestion
    .card-body {
      display: flex;
      flex-direction: column;
      justify-content: space-between;
      }
```
こうすることで、 `margin` を指定しなくてもボタンが下端に寄ります